OpenRA.Mods.Cnc/Graphics/Voxel.cs

```diff
@@ -76,7 +80,7 @@ public float[] TransformationMatrix(uint limb, uint frame)
 			return t;
 		}
 
-		public VoxelRenderData RenderData(uint limb)
+		public ModelRenderData RenderData(uint limb)
```

Comment:
I'm not sure about this: in the voxel file format sections are called limbs, so we can't avoid having a naming clash somewhere.  This seems like a better place to have it than at e.g. `sectionData = new Limb[vxl.LimbCount]`